bin/agentConsole.js

```diff
@@ -27,7 +27,7 @@ var readline = require('readline'),
     iotAgentLib = require('../lib/fiware-iotagent-lib'),
     commandUtils = require('command-node'),
     config = require('../config'),
-    logger = require('fiware-node-logger'),
+    logger = require('logops'),
```

Comment:
This module has been used in all other projects for months, and was working without a problem, and I have used it in the Thinking Things IOTA in recent dates and is changing the levels fine, AFAIK.